Add ESP32 support (#28)
cloud4rpi/docs: docs/start/esp-pio.md

            -D CLOUD4RPI_DEBUG=1

4. Save the `platformio.ini` file and open the `src\main.cpp` file.

> 5. Copy-paste the sample code ([ESP8266](https://github.com/cloud4rpi/cloud4rpi-esp-arduino/blob/master/examples/ESP8266/ESP8266.ino), [ESP32](https://github.com/cloud4rpi/cloud4rpi-esp-arduino/blob/master/examples/ESP32/ESP32.ino)) into it.

Reviewer: ```suggestion
5. Copy-paste the sample code ([ESP8266](https://github.com/cloud4rpi/cloud4rpi-esp-arduino/blob/master/examples/ESP8266/ESP8266.ino), [ESP32](https://github.com/cloud4rpi/cloud4rpi-esp-arduino/blob/master/examples/ESP32/ESP32.ino)).
```

> 7. If the `LED_BUILTIN` constant is not defined in your board's library, change this constant to the pin number to which an LED is connected (for example, `2`).

Reviewer: ```suggestion
7. If the `LED_BUILTIN` constant is not defined in your board's library, change this constant to the LED's pin number (for example, `2`).
```

## Running

6. Hit the `Build` button and wait until the compilation is completed.
8. Hit the `Upload` button.
8. If the upload fails, try to start the board in the flashing mode (connect the power supply with the GPIO0 pin connected to GND) and try uploading again.

> 9. Once flashing is complete, open `Serial Monitor` and watch the sample code's output.

Reviewer: ```suggestion
9. Once flashing is complete, the `Serial Monitor` shows the [device](https://cloud4rpi.io/devices) went online and started to send data
```
